torch/csrc/autograd/variable.cpp

```diff
@@ -53,17 +53,38 @@ IntList Variable::Impl::strides() const {
   return data_.strides();
 }
 
+bool Variable::Impl::is_contiguous() const {
+  AT_ERROR("variable impl does not have is_contiguous");
```

Comment:
you could just call data_.is_contiguous here.

Reply:
Yes, but it's not used, so I wouldn't want to add that unless it's necessary. Who knows what the expected behavior will be when someone does use this in the future.

Reply:
I mean...I don't think there's much room for what is_contiguous means on a variable.